operator/cmd/mesh/operator_test.go

```diff
@@ -82,10 +77,14 @@ func TestOperatorInit(t *testing.T) {
 		},
 	}
 
-	l := clog.NewConsoleLogger(os.Stdout, os.Stderr, installerScope)
-	_, gotYAML, err := renderOperatorManifest(rootArgs, &oiArgs.common)
+	cmd := "operator dump --hub " + oiArgs.common.hub
+	cmd += " --tag " + oiArgs.common.tag
+	cmd += " --operatorNamespace " + oiArgs.common.operatorNamespace
```

Comment:
Nit: suggest to add back the `watchedNamespaces` flag to make sure watching multiple namespaces case is working.